packages/faceted-search/src/components/BasicSearch/BasicSearch.scss

```
@@ -1,26 +1,28 @@
+@use '~@talend/design-tokens/lib/tokens';
+
 .tc-basic-search {
 	align-items: center;
 	display: flex;
 	height: inherit;
 	overflow: visible;
-	padding-left: $padding-large;
-	width: 100%;
 	position: relative;
-	margin-right: 1.5rem;
+	width: 100%;
+	margin-left: tokens.$coral-spacing-s;
+	margin-right: tokens.$coral-spacing-s;
 
 	&::after {
 		content: '';
-		height: 2.5rem;
+		height: tokens.$coral-sizing-xs;
 		width: 0;
-		border-right: 1px solid $gallery;
+		border-right: 1px solid tokens.$coral-color-neutral-border-weak;
```

```suggestion
		border-right: tokens.$coral-border-solid-s tokens.$coral-color-neutral-border-weak;
```
Or something like that

---

Yes sure thanks